NEWS.md

```diff
@@ -1,6 +1,16 @@
 
 # tidyselect (development)
 
+* Selection contexts now interpret the boolean operators specially:
+
+  - `!` negates a selection.
+  - `|` takes the union of two selections.
+  - `&` takes the intersection of two selections.
+
+  These patterns can currently be achieved using `-`, `c()` and
```

Comment:
Probably worth reviewing examples/vignettes/read me and converting to use this form.

Reply:
There's very little documentation in tidyselect, this will need to be done in dplyr.

Reply:
Can you file an issue there so we don’t forget? 

We’ll need to give it a little time to make sure but I suspect we should retire the set based interface.

Reply:
Filed in https://github.com/tidyverse/dplyr/issues/4597